ros/api/common/utils.py

```diff
@@ -64,10 +64,10 @@ def validate_request(*args, **kwargs):
         try:
             data = json.loads(request.data)
         except json.decoder.JSONDecodeError as err:
-            LOG.error('Decoding JSON has failed. %s', repr(err))
+            LOG.error(f"Decoding JSON has failed. {repr(err)}")
             abort(400, message="Decoding JSON has failed.")
         except TypeError as ex:
-            LOG.error('Invalid JSON format. %s', repr(ex))
+            LOG.error(f"Invalid JSON format. {repr(ex)}")
```

Comment:
Shall we add prefixes for this validator function? 
Not a problem if it is static, like `{REQUEST-VALIDATOR}`.

Or you could check in *args/**kwargs if there is a class object or `__name__` and then 
`{type(a).__name__}-VALIDATOR`

I intend to overload this and use for future endpoints.